### DISCUSSION TITLE
How to display multiline docstring on hover?

### DISCUSSION THREAD

**Discussion by ylazy** [BowlerHatLLC/vscode-as3mxml#638] [Q&A] [answered]
My docstring is displaying as single line. How to fix it? I spent a day to find solution but no luck :(

![image](https://user-images.githubusercontent.com/6010183/198961288-6bdd9529-7a73-4d63-a4d7-bf51e20621ca.png)

**Comment by joshtynjala** [comment, marked as answer]
Add a `<p>` and `</p>` HTML tags around new paragraphs (you don't need it for the first paragraph, though).
